src/DependencyInjection/MercureExtension.php

```diff
@@ -70,7 +74,24 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $container->setAlias(Publisher::class, $defaultHub);
+        $container->setAlias(PublisherInterface::class, $defaultHub);
         $container->setParameter('mercure.hubs', $hubUrls);
         $container->setParameter('mercure.default_hub', $defaultHubUrl);
+
+        // Debug & profiler
+        if ($container->getParameter('kernel.debug') && class_exists(Stopwatch::class)) {
```

Comment:
We shouldn't rely on debut (the profiler could be enabled in prod). We should add a new `enable_profiler` configuration option, and set it to true in the recipe for the dev mode.